### PR TITLE
chore: update config.yml; add more community remix official resource links

### PR DESCRIPTION
**Pull request by cliffordfajardo**
## What is the change?
1.  update `config.yml` file to help direct current & potential remix users to the helpful pages

## Motivation
- adds helpful tips for user's during the process of creating a bug/issue report
- bring awareness to other Remix resources to grow the awesome remix community 🤝 📀👥

### CODE REVIEW

**Comment by cliffordfajardo** [comment]
CC @chaance  🙏

**Review by chaance — APPROVED** [review]

**Review by machour — APPROVED** [review]

**Comment by machour** [comment]
@kentcdodds Leaving it up to you to merge this one!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -10,3 +10,12 @@ contact_links:
...
       use the Discussions for this instead of the issues tab 🙂.
+  - name: 💬 Remix Discord Channel
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
+    url: https://twitter.com/remix_run
+    about: Stay up to date with Remix news on twitter
+  - name: 🍿 Remix YouTube Channel
+    url: https://rmx.as/youtube
+    about: Are you a techlead or wanting to learn more about Remix in depth? Checkout the Remix YouTube Channel
```
Comment (chaance) [inline .github/ISSUE_TEMPLATE/config.yml:21]: nit: `techlead` should be `tech lead`

Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/config.yml]: Resolved

**Review by kentcdodds — APPROVED** [review]
Love it 👍

**Pull request merged**